src/test/java/com/programmers/application/service/VoucherServiceImplTest.java

```diff
@@ -0,0 +1,99 @@
+package com.programmers.application.service;
+
+import com.programmers.application.domain.voucher.VoucherType;
+import com.programmers.application.dto.reponse.VoucherInfoResponse;
+import com.programmers.application.dto.request.RequestFactory;
+import com.programmers.application.dto.request.VoucherCreationRequest;
+import com.programmers.application.repository.MemoryVoucherRepository;
+import com.programmers.application.repository.VoucherRepository;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.EnumSource;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+class VoucherServiceImplTest {
+    private static final int FIXED_DISCOUNT_AMOUNT = 100;
+    private static final int PERCENT_DISCOUNT_AMOUNT = 10;
+    private static final String FIXED_AMOUNT_VOUCHER_TYPE = "fixed";
+    private static final String PERCENT_DISCOUNT_VOUCHER_TYPE = "percent";
+
+    private VoucherRepository voucherRepository = new MemoryVoucherRepository();
+    private VoucherService voucherService = new VoucherServiceImpl(voucherRepository);
+
+    @DisplayName("다양한 바우처 생성 성공 테스트")
+    @ParameterizedTest
+    @CsvSource(value = {
+            "FIXED, 100",
+            "PERCENT, 10"})
+    void createVoucher(String voucherType, long discountAmount) {
+        //give
+        VoucherCreationRequest voucherCreationRequest = RequestFactory.createVoucherCreationRequest(voucherType, discountAmount);
+
+        //when
+        UUID voucherId = voucherService.createVoucher(voucherCreationRequest);
+
+        //then
+        Assertions.assertThat(voucherRepository.findByVoucherId(voucherId)).isPresent();
+
+    }
+
+    @DisplayName("잘못된 바우처 타입 입력시 예외 발생 테스트")
```

Comment:
```suggestion
    @DisplayName("잘못된 바우처 타입 입력시 예외가 발생한다.")
```

테스트 이름은 하나의 문장으로 작성해주면 좋습니다. `가정-행위-결과` 로요.